ember-welcome-page/package.json

```diff
@@ -54,6 +54,9 @@
     "@babel/plugin-proposal-decorators": "^7.20.5",
     "@babel/preset-typescript": "^7.18.6",
     "@embroider/addon-dev": "^3.0.0",
+    "@glint/core": "^1.0.0-beta.2",
+    "@glint/environment-ember-loose": "^1.0.0-beta.2",
+    "@glint/template": "^1.0.0-beta.2",
```

Comment:
Note, Glint requires the installed `@glimmer/component` to be `v1.1.2` or higher. Fortunately, the latest version of `ember-cli` already installs `v1.1.2`.

- https://typed-ember.gitbook.io/glint/using-glint/ember/installation#version-requirements
- https://github.com/ember-cli/ember-cli/blob/v4.9.2/blueprints/app/files/package.json#L32

Reply:
`package.json` in `ember-cli@v3.28.6` lists `"@glimmer/component": "^1.0.4"`.

https://github.com/ember-cli/ember-cli/blob/v3.28.6/blueprints/app/files/package.json#L31